.test-infra/jenkins/job_LoadTests_GBK_Flink_Go.groovy

```diff
@@ -27,6 +27,10 @@ import static LoadTestsBuilder.DOCKER_CONTAINER_REGISTRY
 
 String now = new Date().format('MMddHHmmss', TimeZone.getTimeZone('UTC'))
 
+def TESTS_TO_SKIP = [
```

Comment:
You should explain why do you want to skip that test. I think one or two sentences describing the memory issue that you observed is fine. You could also add a TODO and attach it to this JIRA ticket: https://issues.apache.org/jira/browse/BEAM-9761 which tracks problems with GBK and CoGBK for Python (the issue can be possibly related)

Reply:
Ok, I made a comment about it. Also I changed the PR name.